scripts/testing/update_iso

```diff
@@ -217,8 +219,26 @@ def run_updated_iso_with_virt_install():
     subprocess.run(cmd, check=True)
     print("** virt-install finished running")
 
+def set_input_iso(args):
+    """Call checks and configure input ISO based on the user parameter
+
+    This function will configure INPUT_ISO, UPDATED_ISO global variables.
+    """
+    global INPUT_ISO
+    global UPDATED_ISO
+    global INPUT_ISO_REVISION_FILE
+
+    INPUT_ISO = args.input_iso_path
+    UPDATED_ISO = os.path.join(args.input_iso_path.parent,
+                                f"{args.input_iso_path.stem}-updated{args.input_iso_path.suffix}")
+    INPUT_ISO_REVISION_FILE = os.path.join(args.input_iso_path.parent, REVISION_NAME)
+
+
 def main():
-    parser = argparse.ArgumentParser(description="update Anaconda boot.iso")
+    parser = argparse.ArgumentParser(description="update Anaconda ISO image")
+    parser.add_argument('input_iso_path', metavar='INPUT_ISO', type=Path, nargs='?', default=INPUT_ISO,
+                        help="""path to input ISO; this parameter is optional and '-t' parameter
+                        is required when used""")
```

Comment:
I suggest "when providing path to a custom iso" instead of just "when used" to be more clear. :)

Reply:
I removed this part because the `-t` is not always requested. The git_rev is supported even for custom ISO (Live ISO) when present in the same directory.

Reply:
Makes sense - as long as we make sure the remove the revision file when re-generating the ISO, this should work fine. :)